clangFormat/README.md

```diff
@@ -0,0 +1,9 @@
+## Code Style
+The clang format pre-commit hook will execute clang-format on your code after you commit and inform you if there were any style violations.  All files can be found here: [Code Style](https://github.com/Raizlabs/Raizlabs-Cocoa-Style/tree/master/clangFormat)
+
+ - Download the private clang-format binary, copy it into /usr/local/bin/ and chmod +x it.
```

Comment:
#nth explain why we are using a private binary (I think it's because of @KingOfBrian's fix, right?).